src/icon-tab-bar.scss

```diff
@@ -320,6 +344,10 @@ $fd-icon-tab-bar-semantic-values: (
         }
       }
     }
+
+    &--hidden {
+      display: none;
+    }
```

Comment:
where is this used?

Reply:
It is using for overflow directive. You are right, it shouldn't be in styles. Moved to ngx.